jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonAssert.java

```diff
@@ -45,24 +45,6 @@ public ButtonAssert notVisible() {
         return this;
     }
 
```

Comment:
Do you really need all methods in this class?

Reply:
only hasClass method can be deleted, other methods from this class uses other tests